docs/src/main/asciidoc/attributes.adoc

```diff
@@ -5,3 +5,8 @@
 :idseparator: -
 :icons: font
 :imagesdir: ./images
+
+// Product-related attributes
+:ProductName: Quarkus
+:ProductLongName: {ProductName}
+:maven-version: 3.6.3
```

Comment:
@inoxx03 Hey Stefan, hope you don't mind me butting in with this but I wouldn't use the term "product" in the community but would use "downstream" instead. Looking back at Infinispan docs that's one of the things I'd do differently but it creeped in some of the conditionals and attributes.

It's just my 2c but "community" vs "product" is a little distasteful for community users. I would advocate for "downstream" where possible.

Also I think maybe s/ProductName/BrandName - If you want to use ProductName downstream you can always do :BrandName: {ProductName} and then define ProductName separately. It's a little more effort but it keeps things "cleaner" in the community source. You could also use sed to replace attributes downstream. I'd be happy to show you how to set all this up. 

Final thought, if there are attributes that you only need downstream, don't add them here. For Infinispan, we have a separate attributes file that I include in downstream titles. It's better all around to not have to send a PR upstream if you need to modify an attribute value. 

Hope this helps and you don't mind me butting in to your PR.

Reply:
@oraNod got an example on how you do this attribute inclusion work ?

Reply:
@maxandersen sure. For Infinispan, here are community doc attributes: https://raw.githubusercontent.com/infinispan/infinispan/11.0.x/documentation/src/main/asciidoc/topics/attributes/community-attributes.adoc 

Community also has attributes defined in the pom here: https://github.com/infinispan/infinispan/blob/11.0.x/documentation/pom.xml

Downstream uses this file (vpn): https://gitlab.cee.redhat.com/red-hat-jboss-data-grid-documentation/rhdg-documentation/-/blob/8.1.x/rhdg-common/attributes/rhdg-attributes.adoc 

Community and downstream use different title files, for example here's our server guide:

Community title: https://raw.githubusercontent.com/infinispan/infinispan/11.0.x/documentation/src/main/asciidoc/titles/server/server.asciidoc
Downstream title: https://gitlab.cee.redhat.com/red-hat-jboss-data-grid-documentation/rhdg-documentation/-/blob/8.1.x/titles/server/master.adoc

To keep the community docs sync'ed downstream, I use jenkins jobs (it's just bash that runs on a cron). There's a readme that I pull into the server guide that has some `sed` attribute replacing: https://gitlab.cee.redhat.com/rhdg-prod/rhdg-jenkins-csb/-/blob/master/jobs/docs/8.1.x-doc-sync.groovy#L63